Meter.c

```diff
@@ -301,6 +321,27 @@ static void LEDMeterMode_drawDigit(int x, int y, int n) {
 }
 
 static void LEDMeterMode_draw(Meter* this, int x, int y, int w) {
+   assert(x >= 0);
+   assert(w <= INT_MAX - MAXIMUM(0, x));
```

Comment:
```suggestion
   assert(w <= INT_MAX - x);
```

Suffices. `x>=0` has just been checked above.